services/staging/rs_server_staging/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Copyright 2024 CS Group
```

Comment:
What it the purpose of this file ? It is empty except for the header.

Reply:
Not used, will be removed.